### PR TITLE
Refactor routers

### PR DESCRIPTION
**Pull request by fivegrant**

### CODE REVIEW

**Comment by fivegrant** [comment]
@Sorrento110 this moves the qualifier, feature and association endpoints so this could be breaking to data annotation

File: Makefile [hunk 1]
```diff
@@ -4,11 +4,17 @@ PYTHON = $(shell which python3 || which python)
 export LANG
 
 
-.PHONY:init
+.PHONY:
```
Comment (mattprintz) [inline Makefile:7]: The original syntax here was correct: https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html
That is, we for all of the phony targets, we should include the target as a "dependency" of .PHONY.

File: tds/routers/concepts.py [hunk 2]
```diff
@@ -1,32 +1,64 @@
...
+    logger.info("Sending data to %s", url)
+
+    response = requests.get(url, headers=headers, timeout=5)
+    logger.debug("response: %s", response)
+    logger.debug("response reason: %s", response.raw.reason)
+
+    if response.status_code == 200:
+        return json.loads(response.content.decode("utf8"))
+    logger.debug("Failed to fetch ontologies: %s", response)
+    raise Exception("DKG server returned the status {response.status_code}")
```
Comment (mattprintz) [inline tds/routers/concepts.py:61]: Same code in 2 different PRs?

Comment (fivegrant) [reply tds/routers/concepts.py:61]: i merged the code from the other PR into this one since i might write tests

**Review by Sorrento110 — CHANGES_REQUESTED** [review]
I only found one change, I will update data-annotation when this is merged.

**Review by mattprintz — APPROVED** [review]

**Review by Sorrento110 — APPROVED** [review]

**Pull request merged**